packages/components/src/mobile/bottom-sheet/ripple.native.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import {
+	Platform,
+	TouchableOpacity,
+	TouchableNativeFeedback,
+	View,
+} from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { withPreferredColorScheme } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import rippleStyles from './ripple.native.scss';
+
+const ANDROID_VERSION_LOLLIPOP = 21;
+const ANDROID_VERSION_PIE = 28;
+
+const TouchableRipple = ( {
+	style,
+	onPress,
+	disabled: disabledProp,
+	children,
+	activeOpacity,
+	getStylesFromColorScheme,
```

Comment:
Heh, i'm fighting with myself about if it would be wrong to spread not used styles like `accessible` etc.

I know that it is easy to check what prop we can pass if they are listed like that but on the other hand it blocks from passing other props to the Touchable. Let's say i would like to add `hitSlop` or `touchSoundDisabled` to the Ripple. How could i make it? :) I suggest to do something like:

```js
{
  style,
  onPress,
  disabled: disabledProp,
  children,
  getStylesFromColorScheme,
  borderless = false,
  ...touchableProps
}
```

and then spread them in the render to `TouchableNativeFeedback` or `TouchableOpacity`

WDYT? :)

Reply:
Being honest I had the same question when implement this feature and both approach is ok for me. Prop spreading allow to be more flexible with passing additional props as you mentioned and after your comment I think it is better choice in long term so I will go with that solution. Thanks !